lldb/test/API/commands/target/anon-struct/main.cpp

```diff
@@ -0,0 +1,19 @@
+struct A {
+  struct {
+    int x = 1;
+  };
+  int y = 2;
```

Comment:
Is `y` necessary for the reproducer?

Reply:
No; leftover from other tests. Will remove it.